src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -159,38 +165,54 @@ public IEnumerable<SuggestedActionSet> GetSuggestedActions(ISuggestedActionCateg
-                    if (supportSuggestion.SupportsCodeFixes(document) && requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.CodeFix))
-                    {
-                        var suggestions = Task.Run(
-                            async () => await _owner._codeFixService.GetFixesAsync(document, range.Span.ToTextSpan(), cancellationToken).ConfigureAwait(false), cancellationToken).WaitAndGetResult(cancellationToken);
+                    var fixes = GetCodeFixes(
+                        supportSuggestion, requestedActionCategories,
+                        workspace, document, range, cancellationToken);
 
-                        result = OrganizeFixes(workspace, suggestions);
-                    }
+                    var refactorings = GetRefactorings(
+                        optionService, supportSuggestion, requestedActionCategories,
+                        workspace, document, snapshot, range, cancellationToken);
 
-                    if (supportSuggestion.SupportsRefactorings(document))
-                    {
-                        var refactoringResult = GetRefactorings(requestedActionCategories, document, snapshot, workspace, optionService, cancellationToken);
+                    return (fixes == null) ? refactorings :
+                                (refactorings == null) ? fixes : fixes.Concat(refactorings);
+                }
+            }
 
-                        result = result == null ? refactoringResult :
-                                    refactoringResult == null ? result :
-                                        result.Concat(refactoringResult);
+            private IEnumerable<SuggestedActionSet> GetCodeFixes(
+                IDocumentSupportsSuggestionService supportSuggestion,
+                ISuggestedActionCategorySet requestedActionCategories,
+                Workspace workspace,
+                Document document,
+                SnapshotSpan range,
+                CancellationToken cancellationToken)
+            {
+                IEnumerable<SuggestedActionSet> result = null;
+                if (_owner._codeFixService != null && supportSuggestion.SupportsCodeFixes(document))
+                {
+                    var includeSuppressionFixes = requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.Any);
+                    if (requestedActionCategories.Contains(PredefinedSuggestedActionCategoryNames.CodeFix) || includeSuppressionFixes)
+                    {
+                        var suggestions = Task.Run(
+                            async () => await _owner._codeFixService.GetFixesAsync(
+                                includeSuppressionFixes, document, range.Span.ToTextSpan(), cancellationToken).ConfigureAwait(false),
+                            cancellationToken).WaitAndGetResult(cancellationToken);
+                        result = OrganizeFixes(includeSuppressionFixes, workspace, suggestions);
```

Comment:
I was passing this down so that we can avoid traversing the list of fixes a second time inside ```GroupFixes()``` to order suppressions below regular fixes. I agree this is minor and can be removed - but I don't see any harm in keeping this.